app/main.py

```diff
@@ -16,4 +16,19 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+variables = [lucky_number, pi, one_is_a_prime_number, name, my_favourite_films,
+             profile_info, marks, collection_of_coins]
+
+sorted_variables = {
+    "mutable": [],
+    "immutable": []
+}
+
+for each_variable in variables:
+    if (type(each_variable) == list or type(each_variable) == dict
+            or type(each_variable) == set):
```

Comment:
Using `type()` for type checking is not recommended. Consider using `isinstance()` instead. For example, `isinstance(each_variable, (list, dict, set))` would be a more Pythonic way to check if the variable is mutable.